README.md

```diff
@@ -1 +1,50 @@
-# Awsome-books
\ No newline at end of file
+# Awsome-books
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+> This is a simple Book List app Project.
```

Comment:
- [ ] [OPTIONAL] It would be nice to add a description of your project. This might include info on the functionality and use of the project.